bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/FlatRow.java

```diff
@@ -128,13 +128,22 @@ public boolean equals(Object obj) {
         return false;
       }
       Cell other = (Cell) obj;
-      return Objects.equal(family, other.family) &&
-          Objects.equal(qualifier, other.qualifier) &&
-          timestamp == other.timestamp &&
+      return hasEqualKeys(other) &&
```

Comment:
Can you update the class doc to: 

This class stores represents a single row. It's a flattened version of the data of a
  * {@link Row}

Can you confirm that the change in this file is a no-op?

You introduced a concept of the key in this class. Could you mention this in the class doc or at least at the place where it is used.